StyleCop.Analyzers/StyleCop.Analyzers.Test/DocumentationRules/SA1612UnitTests.cs

```diff
@@ -19,16 +20,53 @@ namespace StyleCop.Analyzers.Test.DocumentationRules
     /// </summary>
     public class SA1612UnitTests
     {
+        public static IEnumerable<object[]> DeclarationsWithMemberColumn
+        {
+            get
+            {
+                yield return new object[] { "    public ClassName Method(string foo, string bar, string @new) { return null; }", 22 };
```

Comment:
💡 If these are changed, we should change them to use `{|#0:...|}` syntax for the member column span

Reply:
I might have misunderstood you, but in the cases where a diagnostic is expected, it is not always on the member declaration, so adding them here doesn't sound like it would work. See for example VerifyIncludedMemberWithInvalidParamsIsReportedAsync.

Reply:
A numbered annotation `#N` is simply ignored if there is no corresponding `Diagnostic().WithLocation(N)`

Reply:
Thanks! Updated.